### PR TITLE
Deferred publishing (publishAt) for articles

### PR DESCRIPTION
**Pull request by UladBohdan**
* `getOne` is modified to only return article with is already published (for regular users)
* same for `getAll`
* verified that `create` also populates `publishAt` field. Check out the postman template.

* translation for english errors.

The test I've added is failing - I need help here!

### CODE REVIEW

**Comment by UladBohdan** [comment]
ok I did my best
js guys, you turn!

**Comment by UladBohdan** [comment]
btw
* do we need travis both on every commit and on pull requests?
* do we need codacy? Travis returns the same lint warnings.

**Comment by UladBohdan** [comment]
The bug still exists.
Article tests fail if pageSize query option is not specified / larger than some value. This is true both for the new test and for the one which existed before.

**Comment by Drapegnik** [comment]
[login: to return a general "failed to login" message instead of "wrong email" / "bad password".](https://trello.com/c/AgkAwy5E/80-login-to-return-a-general-failed-to-login-message-instead-of-wrong-email-bad-password)

**Comment by Drapegnik** [comment]
[backend get current user endpoint](https://trello.com/c/7RSGivz6/79-backend-get-current-user-endpoint)

File: src/api/article/controller.js [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+    };
+  }
 
-  return Article
-    .find({})
+  return Article.find(articlesQuery)
     .populate('type')
     .skip(skip)
     .limit(pageSize)
+    .sort({ createdAt: 'desc' }) // FIXME(@uladbohdan): maybe change sort field on publishAt
```
Comment (Drapegnik) [inline src/api/article/controller.js]: @UladBohdan check it!

**Comment by Drapegnik** [comment]
[deferred publication feature](https://trello.com/c/9I9NusfW/77-deferred-publication-feature)

**Pull request merged**